src/runtime/contrib/cublas/cublas.cc

```diff
@@ -124,35 +175,205 @@ struct CublasDgemmBatchOp {
   }
 };
 
+// Check cublas supported mix-precision computation type and return computeType
+bool CheckCublasMixPrecisionType(DLDataType in_dtype, DLDataType out_dtype, bool int_support = true) {
```

Comment:
Maybe shorten the name to just `CheckMixPrecisionType` since the function is under cublas context